RubberduckTests/QuickFixes/RemoveUnassignedVariableUsageQuickFix.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System.Linq;
```

Comment:
not really in the scope of this PR, but the file (and possibly the class) is missing the Test suffix

Reply:
I will add it.

Reply:
Hm, apparantly, renaming a file without changes does not make comments in the original file outdated.